StRoot/StETofUtil/StETofGeometry.cxx

```diff
@@ -600,23 +665,30 @@ StETofGeometry::init( TGeoManager* geoManager, const double* safetyMargins, cons
 
     mNValidModules = 0;
 
+
+	 if( mFileNameAlignParam != ""){
+	 	readAlignmentParameters();
+	 }else{
+		readAlignmentDatabase(maker);
+	 }
+
+	 geoManager->AddNavigator();
+
+	 int iCounterAlignment = 0;
     // loop over sectors
     for( int sector = eTofConst::sectorStart; sector <= eTofConst::sectorStop; sector++ ) {
         // loop over planes
         for( int plane = eTofConst::zPlaneStart; plane <= eTofConst::zPlaneStop; plane++ ) {
             std::string geoPath( formTGeoPath( geoManager, plane, sector ) );
 
             if( geoPath.empty() ) {
-                LOG_DEBUG << "StETofGeometry::Init(...) - cannot find path to ETOF module "
+                LOG_INFO << "StETofGeometry::Init(...) - cannot find path to ETOF module "
                             "(id " << plane << sector << "). Skipping..." << endm;
                 continue;
             }
             mNValidModules++;
-
```

Comment:
Please don't make so many changes in white space. This is completely irrelevant to the primary goal of this PR

Reply:
Noted :)